learning/step_by_step/your_first_game.rst

```diff
@@ -225,6 +225,12 @@ more fast diagonal movement.
 We also check whether the player is moving so we can start or stop the
 AnimatedSprite animation.
 
+.. tip:: ``$`` returns the node at the relative path from this node, or will return ``null`` if the node is not found.
+         Since AnimatedSprite is a child of this node, we can just use ``$AnimatedSprite``.
```

Comment:
mixed tense: "returns" vs. "will return". Change the latter to "returns ``null``"

It may be unclear what "this node" refers to. I would say "the current node"